README.md

```diff
@@ -100,18 +104,18 @@ Finally, with given scenarios and multi-partner learning approaches, we can addr
        epoch_count: 
          - 38
        methods:
-         - ["Shapley values", "Independent scores", "TMCS"]
+         - ["Shapley values", "SMCS", "IS_lin_S", "IS_reg_S"]
        minibatch_count: 
          - 20
        gradient_updates_per_pass_count:
          - 8
     ```
 
-   Under `scenario_params_list`, enter a list of sets of scenario(s). Each set starts with ` - partners_count:` and must have only one `partners_count` value. The length of `amount_per_partners`, `corrupted_datasets` (and `samples_split_option` when the advanced definition is used) must match the `partner_counts` value. If for a given parameter multiple values are specified, e.g. like for `agregation_weighting` in the first scenario set of the above example, all possible combinations of parameters will be assembled as separate scenarios and run.
+   Under `scenario_params_list`, enter a list of sets of scenario(s). Each set starts with ` - dataset_name:` and must have only one `partners_count` value. The length of `amount_per_partners`, `corrupted_datasets` (and `samples_split_option` when the advanced definition is used) must match the `partner_counts` value. If for a given parameter multiple values are specified, e.g. like for `agregation_weighting` in the first scenario set of the above example, all possible combinations of parameters will be assembled as separate scenarios and run.
    
 3. Then execute `main.py -f config.yml`
 
-4. A `results.csv` file will be generated in a new folder for your experiment under `/experiments/<your_experiment>`. You can read this raw `results.csv` file or use the `analyse_results.ipynb` generated notebook to quickly generate figures.
+4. A `results.csv` file will be generated in a new folder for your experiment under `/experiments/<your_experiment>`. You can read this raw `results.csv` file or use the generated notebooks in `/notebooks`.
```

Comment:
or use the notebooks in `/notebooks`.

The notebooks are not re-generated at every experiment

Reply:
Ok